frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -228,11 +228,14 @@ const removeTrigger: KebabAction = (kind: K8sKind, pipeline: Pipeline) => ({
     verb: 'delete',
   },
 });
-export const getPipelineKebabActions = (pipelineRun?: PipelineRun): KebabAction[] => [
+export const getPipelineKebabActions = (
```

Comment:
can we add tests for **getPipelineKebabActions** utils?

Reply:
Added tests for this utils.